Makefile

```diff
@@ -26,4 +45,7 @@ lint:
 	black --check actions
 
 types:
-	pytype --keep-going actions
\ No newline at end of file
+	pytype --keep-going actions
```

Comment:
On that note ^ 
`pytype --keep-going actions tests`

as well as `black actions tests` and `flake8 actions tests`